Sources/DatadogSDKTesting/Utils/PlatformUtils.swift

```diff
@@ -88,6 +88,10 @@ struct PlatformUtils {
             return (ProcessInfo.processInfo.processName, (Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String) ?? "")
         }
     }
+    
+    static func getCpuCount() -> Int {
+        ProcessInfo.processInfo.processorCount
```

Comment:
Is this CPU or vCPU count? Is there a diff in Swift?

Reply:
It is vCPU. Apple ARM devices don't support hyper-threading so vCPU == CPU cores. For Intel it will show 2x CPU cores.